bin/fmt.ml

```diff
@@ -0,0 +1,29 @@
+open Stdune
+open Import
+
+let doc = "Format source code"
+
+let man =
+  [ `S "DESCRIPTION"
+  ; `P
+      {|$(b,dune fmt) runs the formatter on the source code and promotes the result.|}
```

Comment:
We don't want to mention promotion because it's an implementation detail. One that isn't very desirable and we intend to change it.